source/index.html.md.erb

```diff
@@ -1,16 +1,26 @@
 ---
 title: GOV.UK One Login Technical Documentation
 weight: 1
-last_reviewed_on: 2022-09-06
+last_reviewed_on: 2023-01-24
 review_in: 6 months
 ---
 
 # GOV.UK One Login
 
-GOV.UK One Login is the new way for government services to sign in their users.
+GOV.UK One Login is the way for government services to:
 
-GOV.UK One Login will allow users to access any government service using the same username and password.
+* sign in their users
+* prove their users' identity
+
+This technical documentation will help you understand how GOV.UK One Login works. It gives you information on how you:
```

Comment:
this tech doc will help you understand how you can use GOV.UK One Login?

Reply:
I do wonder if it does need something more specific. "This technical documentation tells you how to use GOV.UK One Login to xxxx [something in summary of what this page is helping people with, if possible?]

Reply:
```suggestion
This technical documentation gives you information on how to:
```

Perhaps we can just simplify it and shorten it to this?